djvm/src/main/kotlin/net/corda/djvm/rewiring/ClassRewriter.kt

```diff
@@ -45,7 +47,7 @@ open class ClassRewriter(
             configuration.definitionProviders,
             configuration.emitters
         )
-        visitor.analyze(reader, context, options = ClassReader.EXPAND_FRAMES)
+        visitor.analyze(reader, context, options = SKIP_FRAMES)
```

Comment:
Could you please explain the usage of COMPUTE_FRAMES and SKIP_FRAMES and why it has become necessary?

Reply:
This is an ASM option. Basically, I am telling its "reader" not to bother processing any stack frames that it finds because I've told the "writer" to compute brand new stack frames from scratch.